sql/core/src/main/java/org/apache/spark/sql/execution/UnsafeKVExternalSorter.java

```diff
@@ -44,6 +45,7 @@
  */
 public final class UnsafeKVExternalSorter {
 
+  private static final int DEFAULT_INITIAL_SORT_BUFFER_SIZE = 4096;
```

Comment:
We have this constant repeated in 3 places. I think 2 of them can be combined -- the ones in the same package at least.